data/discipline-easy-push-20221026.md

```diff
@@ -0,0 +1,33 @@
+# discipline workout placeholder title
```

Comment:
Out of curiosity, is `placeholder title` part of the workout's name? In case it isn't please remove that part